src/storage_types.rs

```diff
@@ -125,6 +132,8 @@ macro_rules! storage_types {
         storage_types!(@type ($(#[$attr])*) @$M Rational32 ($($tt)*));
         storage_types!(@type ($(#[$attr])*) @$M Rational64 ($($tt)*));
         storage_types!(@type ($(#[$attr])*) @$M BigRational ($($tt)*));
+        storage_types!(@type ($(#[$attr])*) @$M Complex32 ($($tt)*));
+        storage_types!(@type ($(#[$attr])*) @$M Complex64 ($($tt)*));
```

Comment:
This change adds a new type alias, `VV` for complex underlying storage types. e.g. `f32` for `Complex32` and `f64` for `Complex64`.

```suggestion
        storage_types!(@type ($(#[$attr])*) @$M Complex32 (#[allow(dead_code)]
            pub type VV = f32;
            $($tt)*));
        storage_types!(@type ($(#[$attr])*) @$M Complex64 (#[allow(dead_code)]
            pub type VV = f64;
            $($tt)*));
```